src/main/java/use_cases/login_leaderboard/IRegisterUserOutputBoundary.java

```diff
@@ -0,0 +1,11 @@
+package use_cases.login_leaderboard;
+
+
+/***
+ * The output boundary for the register use case that helps display whether
+ * the registration was successful or not.
+ */
+public interface IRegisterUserOutputBoundary {
+
+    void PrepareView(String u);
```

Comment:
since the implementation of this leaves the method caller to decide the text to display maybe move this somewhere more classes can access it... or make RegisterUserPresenter not only for the user registration section (if a login was successful or not, if a new maze was created successfully or not etc.)

Reply:
I'm still thinking about the design for the login case. I think it would be cleaner if each of those cases had their own different Presenters, since it's just a couple lines of code